### DISCUSSION TITLE
Failed to boot from Teslaandroid image from USB SSD

### DISCUSSION THREAD

**Discussion by Get-108** [tesla-android/issue-tracker#288] [Q&A] [answered]
Hello!
Failed to load boot.scr error after starting booting.
– USB boot is activated, EEPROM and bootloader are up to date.
– Tried: write image via raspberry imager, etcher, copying from working SD via dd command.
Nothing works.

Booting from SD is ok.
What i do wrong? Thank you for any advice.

**Comment by mikegapinski** [comment, marked as answer]
USB boot is not supported

**Reply by apple2ms** [reply]
Is there any updated instructions on how to do that with the nvme ?

**Reply by mikegapinski** [reply]
> Is there any updated instructions on how to do that with the nvme ?

Nvme on the CM4?

**Reply by apple2ms** [reply]
No, the pi4. I saw you are now offering one with nvme and I was wondering what kind of hardware it uses, assumed pi4 with usb nvme

**Comment by mikegapinski** [comment]
NVMe via USB is not the same as NVMe via PCIe on the CM4. It is not supported, completely different boot method

Od: apple2ms ***@***.***>
Data: poniedziałek, 23 października 2023 o 19:52
Do: tesla-android/issue-tracker ***@***.***>
DW: Michał Gapiński ***@***.***>, Comment ***@***.***>
Temat: Re: [tesla-android/issue-tracker] Failed to boot from Teslaandroid image from USB SSD (Discussion #288)

No, the pi4. I saw you are now offering one with nvme and I was wondering what kind of hardware it uses, assumed pi4 with usb nvme
